src/PrestaShopBundle/Controller/Admin/TranslationsController.php

```diff
@@ -194,11 +196,19 @@ protected function findLanguageByLocale($locale)
     protected function getTranslationsCatalogue(Request $request)
     {
         $lang = $request->request->get('lang');
+        $type = $request->request->get('type');
 
         $translator = $this->container->get('translator');
 
         $locale = $this->langToLocale($lang);
 
+        $tree = $this->get('ps.translations_factory')->createTree($type, $locale);
```

Comment:
https://github.com/mickaelandrieu/PrestaShop/blob/76f83eaf98ef5f94422adcd0d02f49b8664e4079/src/PrestaShopBundle/Controller/Admin/TranslationsController.php#L62 is not a catalogue, but already an associative array :)

I agree to rename the function ``createTree`` and the variable name, but not to ``catalogue`` because it's **not** a ``MessageCatalogue``.

How about ``createTranslationsArray`` and ``$translations`` as variable name ?